src/lib/eth.ts

```diff
@@ -18,19 +17,20 @@ export type EthereumWindow = Window & {
   ethereum?: Provider
 }
 
-export async function createEth() {
+export async function getProvider(
```

Comment:
What do you think of splitting this method into two? 

`getConnectedProvider` which receives 0 params and return the current connected provider (OLD changed `getProvider()`. And `createProvider` or `getCustomProvider` or `createCustomProvider` or whatever name you fee comfortable which receives `chainId` and `providerType` and returns a provider created by dcl-connect: `connection.createProvider(providerType chainId)`

Reply:
love it, let me take a look at the implementation and report back if I see any problems